onyo/lib/tests/test_commands_new.py

```diff
@@ -159,7 +161,7 @@ def test_onyo_new_creates_directories(inventory: Inventory) -> None:
 
     # new assets are added
     for s in specs:
-        p = new_directory / inventory.generate_asset_name(s)
+        p = new_directory / inventory.generate_asset_name(Item(s, repo=inventory.repo))
```

Comment:
I would suggest to use this pattern instead:

```suggestion
        p = new_directory / inventory.generate_asset_name(inventory.get_item(s))
```

Reply:
Obviously not really important. But I feel it's easier/smoother reading.

Reply:
Much smoother reading. I saw this in your other PR.  Fixed. :-)

Reply:
Actually, I take it back. In this case, we should be doing the awkward thing.

`s` is a spec, and `inventory.get_item()` accepts a `Path`. The point of this entire assignment is to build the path of the asset (that we don't have yet).

So in this case, we're having `generate_asset_name` build the name for us.

In the end, this is all the wrong way to test whether the asset exists, but I don't want to address that in this PR.